packages/components/src/components/domain/unifiedSearch/useUnifiedSearch.ts

```diff
@@ -290,7 +288,7 @@ export class UnifiedSearch {
   modifyFilters(
     search: Partial<UnifiedSearchParameters>,
     transitionOptions?: TransitionOptions
-  ) {
+  ): void {
```

Comment:
Should be needless to define the return type here

Reply:
Removed as unnecessary as "yarn typecheck" passes still without this.